.travis.yml

```diff
@@ -20,3 +22,10 @@ after_success:
 notifications:
   irc:
   - "chat.freenode.net#wet-boew"
+
+cache:
+  bundler: true
+  directories:
+  - node_modules
+  - lib
+  - vendor/cache
```

Comment:
You can see it here https://travis-ci.org/wet-boew/wet-boew/caches, or by selecting "Caches" under the Settings dropdown